core/src/main/java/de/evoila/cf/broker/configuration/RabbitOnConditionalConfiguration.java

```diff
@@ -49,24 +49,4 @@ public RabbitTemplate rabbitTemplate() {
             }
 
       }
-
-      @Configuration
-      @Profile("cloud")
-      static class Cloud extends AbstractCloudConfig {
```

Comment:
Why was the cloud config removed? How should that work when deployed to CF?

Reply:
Auto configuration is sufficient for the cloud. This Code is not needed.